module/dkg/broker.go

```diff
@@ -415,31 +415,36 @@ func (b *Broker) listen() {
 // onPrivateMessage verifies the integrity of an incoming message and forwards
 // it to consumers via the msgCh.
 func (b *Broker) onPrivateMessage(originID flow.Identifier, msg messages.DKGMessage) {
-	err := b.checkMessageInstanceAndOrigin(msg)
-	if err != nil {
-		b.log.Err(err).Msg("bad message")
-		return
-	}
-	// check that the message's origin matches the sender's flow identifier
-	nodeID := b.committee[msg.Orig].NodeID
-	if !bytes.Equal(nodeID[:], originID[:]) {
-		b.log.Error().Msgf("bad message: OriginID (%v) does not match committee member %d (%v)", originID, msg.Orig, nodeID)
+	if memberIndex, ok := b.committee.GetIndex(originID); ok {
```

Comment:
We can invert this conditional to keep the main logic flow at the top level of indentation.

```
memberIndex, exists := b.committee.GetIndex(originID)
if !exists {
    // log error
    return
}

// happy-path logic
// ... 
```

Reply:
https://github.com/onflow/flow-go/pull/2238/commits/7a287cfa9c3d71c3b4f593d01f47ba48e182dd29